runtime/pom.xml

```diff
@@ -86,7 +92,7 @@
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
-                            <outputDirectory>${project.build.directory}/runtime</outputDirectory>
+                            <outputDirectory>${project.build.directory}</outputDirectory>
```

Comment:
By the way, I think it's better to keep the resources in `runtime` folder, else we have all jar files directly in `target` folder.

Reply:
reverted this change.